docs/app-check/usage/index.md

```diff
@@ -89,6 +89,37 @@ For instructions on how to generate required keys and register an app for the de
 
 You must call initialize the AppCheck module prior to calling any firebase back-end services for App Check to function.
 
+
+
+
+#### Configure AppCheck with iOS credentials (react-native 0.77+)
+
+To do that, edit your `ios/ProjectName/AppDelegate.swift` and add the following two lines:
+
+At the top of the file, import the FirebaseCore SDK right after `import UIKit`:
+And within your existing `didFinishLaunchingWithOptions` method, add the following to the top of the method:
+
+```diff
+import UIKit
++ import RNFBAppCheck
++ import FirebaseCore
+import React
+import React_RCTAppDelegate
+import ReactAppDependencyProvider
+- import Firebase //You can remove this line if upgrading to 0.77+
```

Comment:
I'd prefer to be declarative about positive things then try to handle all negative / don't do items

```suggestion
```